Project/Source Files/Services/Networking/Recipes/RecipeRequester.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  RecipeRequester.swift
+//  NetguruCollegeApp
+//
+
+import Foundation
+
+/// Requester for Recipe Puppy API
+struct RecipeRequester {
+    private let apiClient: APIClient
+    init(with apiClient: APIClient) {
+        self.apiClient = apiClient
+    }
+    func fetchRecipes(for ingredients: [String], handler: @escaping (Recipes) -> Void) {
+        let request = RecipeRequest(with: ingredients)
+        apiClient.sendRequestAndDecode(
+            request: request,
+            success: { (data: Recipes) in
+                handler(data)
+            }) { error in
+                print("Something went wrong: \(error!.localizedDescription)")
```

Comment:
Are you sure that `error!` will never fail? It can cause a crash in the whole app.
Also, printing error is not the best practice. What we could do is to pass and error forward and show an alert with an error to the user.